bot_sample.go

```diff
@@ -74,24 +72,6 @@ func main() {
 
 	webSocketClient.Listen()
 
-	ticker := time.NewTicker(5 * time.Second)
-	defer ticker.Stop()
-
-	go func() {
-		for {
-			select {
-			case resp := <-webSocketClient.EventChannel:
-				HandleWebSocketResponse(resp)
-			case <-ticker.C:
-				webSocketClient.Conn.SetWriteDeadline(time.Now().Add(2 * time.Second))
-				if err := webSocketClient.Conn.WriteMessage(websocket.PingMessage, []byte{}); err != nil {
-					println("send heartbeat error: ", err)
-					return
-				}
```

Comment:
Hi @saf6260 ! It's only the ping part which is not required. The handling of the eventChannel can still be done.

Reply:
Ah, thank you for catching that! My mistake, the websocket event handler should be back